target/linux/ramips/dts/mt7628an_mediatek_linkit-smart-7688.dts

```diff
@@ -144,6 +138,15 @@
 	};
 };
 
+&gpio {
+	bootstrap {
+		gpio-hog;
+		line-name = "bootstrap";
+		output-low;
+		gpios = <11 GPIO_ACTIVE_HIGH>;
```

Comment:
It's better to use `GPIO_ACTIVE_LOW`, though the GPIO status is controlled by `output-low`/`output-high` property.

Reply:
right, I was confused by this. Other usages of gpio-hog are inconsistent as well.